diff -urN enblend-enfuse-4.0-753b534c819d/src/Makefile.in enblend-enfuse-4.0-753b534c819d-patched/src/Makefile.in
--- enblend-enfuse-4.0-753b534c819d/src/Makefile.in	2009-12-21 01:40:27.000000000 +1000
+++ enblend-enfuse-4.0-753b534c819d-patched/src/Makefile.in	2012-08-27 21:04:23.191811459 +1100
@@ -335,7 +335,7 @@
 	-test -z "$(bin_PROGRAMS)" || rm -f $(bin_PROGRAMS)
 enblend$(EXEEXT): $(enblend_OBJECTS) $(enblend_DEPENDENCIES) 
 	@rm -f enblend$(EXEEXT)
-	$(enblend_LINK) $(enblend_OBJECTS) $(enblend_LDADD) $(LIBS)
+	$(enblend_LINK) $(enblend_OBJECTS) $(enblend_LDADD) $(LIBS) -lboost_system
 enfuse$(EXEEXT): $(enfuse_OBJECTS) $(enfuse_DEPENDENCIES) 
 	@rm -f enfuse$(EXEEXT)
 	$(enfuse_LINK) $(enfuse_OBJECTS) $(enfuse_LDADD) $(LIBS)
